Mage.Sets/src/mage/cards/l/LilyBowenRagingGrandma.java

```diff
@@ -0,0 +1,111 @@
+package mage.cards.l;
+
+import java.util.UUID;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.BeginningOfUpkeepTriggeredAbility;
+import mage.abilities.common.EntersBattlefieldAbility;
+import mage.abilities.condition.common.SourceMatchesFilterCondition;
+import mage.abilities.decorator.ConditionalOneShotEffect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.DoubleCountersSourceEffect;
+import mage.abilities.effects.common.GainLifeEffect;
+import mage.abilities.effects.common.counter.AddCountersSourceEffect;
+import mage.constants.*;
+import mage.abilities.keyword.VigilanceAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.counters.CounterType;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.mageobject.PowerPredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+
+/**
+ * @author Cguy7777
+ */
+public final class LilyBowenRagingGrandma extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent();
+
+    static {
+        filter.add(new PowerPredicate(ComparisonType.FEWER_THAN, 17));
```

Comment:
An alternative predicate is `PowerPredicate(ComparisonType.OR_LESS, 16)`
It is equivalent to yours, but matchs better with the card's text. Not a big deal.

Reply:
Good point, thanks!